app/routes/_gcn.circulars.edit.$circularId/CircularEditForm.tsx

```diff
@@ -133,10 +135,30 @@ export function CircularEditForm({
   const [showPreview, setShowPreview] = useState(false)
   const sending = Boolean(useNavigation().formData)
   const valid = subjectValid && bodyValid
+  let headerText,
+    saveButtonText = ''
```

Comment:
```suggestion
  let headerText, saveButtonText
```

Reply:
It's automatically initializing `saveButtonText` to an empty string when it is defined by all paths through the switch statement? Why? Prettier normally doesn't make semantic changes like that.

Reply:
oh i thought this was a formatting change, nevermind